src/scenes/Header/ProfileToolbar/ProfileToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+import { makeStyles, Typography } from '@material-ui/core';
+import { AccountCircle, MoreVert, NotificationsNone } from '@material-ui/icons';
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+import * as React from 'react';
+import { useSession } from '../../../components/Session/Session';
+import { ProfileMenu } from '../ProfileMenu/ProfileMenu';
+
+const useStyles = makeStyles(({ palette, typography }) => ({
+  name: {
+    fontWeight: typography.fontWeightMedium,
+  },
+  menuWrap: {
+    position: 'relative',
+  },
+  infoWrap: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+    backgroundColor: 'white',
+    boxShadow: '0px 9px 12px rgba(209, 218, 223, 0.43)',
+    width: '250px',
```

Comment:
Hardcoding `width` isn't a good idea here, since name is a dynamic length. This is an example of where we, as devs, have read between the lines of the mock ups.
We could:
- Use `minWidth` instead so if the name is longer than expecting we can stretch to it.
- Leave off the extra spacing
- Add a left margin/padding to create that illusion of extra space we aren't filling up.

Reply:
I vote leave off the extra spacing 🤷